README.rst

```diff
@@ -281,8 +281,12 @@ avoid conflicting options. Here follows the example:::
         'password': '123',
         'service_name': 'master',
         'socket_timeout': 0.1,
+        'retry_period': 60,
     }
 
+If ``retry_period`` is given, retry connection up to 3 times in ``retry_period`` intervals. If not set, retrying mechanism is not triggered.
```

Comment:
Why limit to 3 times? Should it also be configurable and/or support infinite retries?